modules/ossm-migrating-workloads-using-the-istio-revision-label-with-cert-manager.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-revision-label-with-cert-manager_{context}"]
+= Migrating workloads using the Istio revision label with cert-manager
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
```

Comment:
Is this a new feature? I was thinking that "Now" seems odd here.

Reply:
It's not a new feature. The intent was to create a guide where we walk customers through an elaborate migration. Service Mesh 3 is a brand new product, and it is not compatible with Service Mesh 2. So the word now is to serve as a reaffirming statement that you can migrate the workloads - meaning you've done the required stuff before you can do this.  It's late...and the release deadline is Monday. Now can be deleted.